utilities_common/sfp_helper.py

```diff
@@ -0,0 +1,53 @@
+import ast
+
+QSFP_DATA_MAP = {
```

Comment:
why are we then keeping QSFP_DD_DATA_MAP {} in sfputil? can we move that also here?

Reply:
Hi @prgeor , QSFP_DD_DATA_MAP is not related to my change. This PR is aimed to fix the format of Application Advertisement. Moving QSFP_DD_DATA_MAP is part of refactoring sfputi/main.py and sfpshow, I suppose it is out of the scope of this PR. 

Moving similar logic of sfputil/main.py and sfpshow to a common place is a good idea, we shall do it in another PR.